dev/accordion.html

```diff
@@ -0,0 +1,136 @@
+            padding: 1rem;
+            opacity: 1;
+            transform: translateY(0);
+        }
+
+        .container.ac .acc-title {
+            background-color: #ccc;
+        }
+
+        .container.ac {
+            border-color: #aaa;
+            box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+        } 
+    </style>
+</head>
+<body>
+    <aalam-accordion id="accordion" activecls="ac">
+        <div class="container ac">
+            <div class="acc-title">
+                <span>What is Artificial Intelligence?</span>
+                <button>Toggle</button>
+            </div>
+            <div class="acc-body">
+                <p>
+                    Artificial Intelligence (AI) is the simulation of human
+                    intelligence processes by machines, especially computer
+                    systems. These processes include learning (the
+                    acquisition of information and rules for using the
+                    information), reasoning (using rules to reach
+                    approximate or definite conclusions), and
+                    self-correction.
+                </p>
+            </div>
+        </div>
+        <div class="container ac">
+            <div class="acc-title">
+                <span>What is Machine Learning?</span>
+                <button>Toggle</button>
+            </div>
+            <div class="acc-body">
+                Machine Learning is a subset of AI that provides systems the
+                ability to automatically learn and improve from experience
+                without being explicitly programmed. It focuses on the
+                development of computer programs that can access data and   
+                use it to learn for themselves.
+            </div>
+        </div>
+        <div class="container">
+            <div class="acc-title">
+                <span>What is Blockchain?</span>
+                <button>Toggle</button>
+            </div>
+            <div class="acc-body">
+                Blockchain is a decentralized, distributed ledger technology
+                that records the provenance of a digital asset. By
+                decentralizing the data, blockchain eliminates the need for
+                intermediaries, reduces the risk of fraud, and enhances
+                transparency.
+            </div>
+        </div>
+    </aalam-accordion>
+    <div>
+        <label>
+            <input type="checkbox" id="nocloseall"> No Close All
+        </label>
+        <label>
+            <input type="checkbox" id="nomultiple"> No Multiple
+        </label>
+    </div>
+    <script>
+        const accordion = document.getElementById('accordion');
+        const noCloseAllCheckbox = document.getElementById('nocloseall');
+        const noMultipleCheckbox = document.getElementById('nomultiple');
+
+        noCloseAllCheckbox.addEventListener('change', (event) => {
+            accordion.setAttribute('nocloseall', event.target.checked);
```

Comment:
You need to remove the attriubte when the target.checked is false. Because boolean attributes are true as long as they are present.